jsonschema-module-jackson/src/test/java/com/github/victools/jsonschema/module/jackson/JacksonModuleTest.java

```diff
@@ -292,4 +315,16 @@ public Long getFieldWithDescriptionAndOnGetter() {
             return fieldWithDescriptionAndOnGetter;
         }
     }
+
+    private static class TestClassWithRequiredAnnotatedFields {
+        @JsonProperty(required = true)
+        private String requiredTrue;
+
+        @JsonProperty(required = false)
+        private String requiredFalse;
+
```

Comment:
We can also include the case where the `required` field is not set, but the annotation is present.
```suggestion

        @JsonProperty
        private String requiredDefault;

```